Doc/whatsnew/3.8.rst

```diff
@@ -295,6 +295,24 @@ Deprecated
   versions. :class:`~ast.Constant` should be used instead.
   (Contributed by Serhiy Storchaka in :issue:`32892`.)
 
+* Functions :func:`~gettext.lgettext`, :func:`~gettext.ldgettext`,
```

Comment:
```suggestion
* The following functions are deprecated: :func:`~gettext.lgettext`, :func:`~gettext.ldgettext`,
```

Reply:
How to conform it with the ending of this sentence? "... in the gettext module and corresponding methods of the NullTranslations and GNUTranslations classes."

Reply:
I think it reads okay, but GitHub sure doesn't make it easy to review.  I wish I could just expand the file at that position.